README.md

```diff
@@ -1850,6 +1852,72 @@ Server: spray-can/1.3.3
 }
 ```
 
+## POST /api/workflows/:version/:id/call-caching
+
+This endpoint allows for reconfiguration of call cache result reuse settings for all calls within a workflow.
+
+Accepted parameters are:
+
+<ul>
+  <li>`allow` Mandatory boolean value, specifies whether call cache result reuse is allowed for all calls in the
+   specified workflow.</li>
+</ul>
+
+cURL:
+
+```
+$ curl -X POST http://localhost:8000/api/workflows/v1/e442e52a-9de1-47f0-8b4f-e6e565008cf1/call-caching?allow=false
+```
+
+HTTPie:
+
+```
+$ http POST http://localhost:8000/api/workflows/v1/e442e52a-9de1-47f0-8b4f-e6e565008cf1/call-caching?allow=false
+```
+
+Response:
+```
+HTTP/1.1 200 OK
+Content-Length: 1
+Content-Type: application/json; charset=UTF-8
+Date: Thu, 04 Jun 2015 12:15:33 GMT
+Server: spray-can/1.3.3
+
+```
+
+## POST /api/workflows/:version/:id/call-caching/:call
```

Comment:
Yes I should document that.  Currently it is ".0", and an absence of index on a scattered call is currently interpreted as a reference to a collector and is disallowed.  But interpreting that situation as "all shards of the scattered call" might be nice.